apps/ecommerce/frontend/src/components/App/Field/SingleResource/SingleResource.tsx

```diff
@@ -1,11 +1,36 @@
-import { /* useCMA, */ useSDK } from '@contentful/react-apps-toolkit';
+import { useEffect, useState } from 'react';
+import { useCMA, useSDK } from '@contentful/react-apps-toolkit';
 import { FieldAppSDK } from '@contentful/app-sdk';
 import { ResourceField } from 'components/App/Field/ResourceField';
-import mockValue from 'helpers/mockValue';
 import ResourceFieldProvider from 'providers/ResourceFieldProvider';
+import { config } from 'config';
+import fetchWithSignedRequest from 'helpers/signedRequests';
 
 const SingleResource = () => {
+  const [logoUrl, setLogoUrl] = useState<string>('');
+
   const sdk = useSDK<FieldAppSDK>();
+  const cma = useCMA();
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const url = new URL(`${config.backendApiUrl}/api/config.json`);
+
+        const res = await fetchWithSignedRequest(url, sdk.ids.app!, cma, sdk, 'GET');
+
+        if (!res.ok) {
+          throw new Error(res.statusText);
+        }
+
+        const data = await res.json();
+
+        setLogoUrl(data.logoUrl);
+      } catch (error: any) {
+        console.error(error.message);
+      }
+    })();
+  }, [sdk, cma]);
```

Comment:
more of a larger thing to discuss, nonblocking, but it seems kind of funky to me that we have this logic into two places, for Single and Multiple resources - is there potentially a way to put this at an even higher level?